packages/polar/sample-project/test/sample-test.js

```diff
@@ -0,0 +1,23 @@
+const { expect, use } = require("chai");
+const { Contract, getAccountByName, polarChai } = require("secret-polar");
+
+use(polarChai);
```

Comment:
why do we use this line ? does user needs to include it everytime ?

Reply:
yes, this is required here. Other option was to add this in the library, but there the path of chai module was to be defined as to separate it from chai used for testing the library. One more way is to use this line in polar config as like used in hardhat but this is cleaner.